libnavui-maneuver/build.gradle

```diff
@@ -68,6 +68,10 @@ dokkaHtmlPartial {
     }
 }
 
+tasks.withType(Test) {
+    android.sourceSets.main.res.srcDirs += 'src/test/res'
+}
+
```

Comment:
Is this necessary? I think it affects not only tests. `android` extension is a global one, it is not specific to `Test` tasks. You should have probably fixed imports in tests. E.g. `import com.mapbox.navigation.ui.maneuver.test.R` instead of `import com.mapbox.navigation.ui.maneuver.R`.

Reply:
Thanks. For some reason I thought that "test" was only added ti androidTest apks.